### ISSUE TITLE
Evaluating after Finetuning on Custom Datasets

### ISSUE THREAD

**Issue by emmaking-smith** [OpenGVLab/InternVL#387]
Hello,

Thank you for all your hard work.  I would like to finetune InterVL Chat V1.2 on a custom dataset, for which there is already a script (https://github.com/OpenGVLab/InternVL/blob/main/document/How_to_finetune_internvl_chat_v1_2_on_a_custom_dataset.md).  

However, the evaluation seems to be dataset-specific.  Is there a script somewhere for evaluation of a custom finetuned model on a custom test set?  I have finetuned with LoRA.

I am specifically looking to evaluate VQA style question-image pairs.

Thank you

**Comment by czczup** [comment]
Hi,

You can refer to our latest documentation for fine-tuning the InternVL2 series models: [https://internvl.readthedocs.io/en/latest/internvl2.0/finetune.html](https://internvl.readthedocs.io/en/latest/internvl2.0/finetune.html).

Then, refer to the evaluation documentation to evaluate the models (if you are fine-tuning with LoRA, you might encounter some issues; feel free to provide feedback): [https://internvl.readthedocs.io/en/latest/internvl2.0/evaluation.html](https://internvl.readthedocs.io/en/latest/internvl2.0/evaluation.html).
